lint-tasks.yml

```diff
@@ -97,6 +136,46 @@ tasks:
           black --color --line-length 100 {{.BLACK_FLAGS}} .
           ruff check {{.RUFF_FLAGS}} .
 
+  nodejs-18:
```

Comment:
We could deduplicate this into a parameterized task in the main Taskfile and then use it in this file with the syntax `:task-name`. See the tip at the end of [this section](https://taskfile.dev/usage/#calling-another-task).